packages/@azure/arm-storage/lib/storageManagementClientContext.ts

```diff
@@ -16,9 +16,15 @@ const packageName = "@azure/arm-storage";
 const packageVersion = "1.0.0-preview";
 
 export class StorageManagementClientContext extends msRestAzure.AzureServiceClient {
+
   credentials: msRest.ServiceClientCredentials;
+
   subscriptionId: string;
 
+  acceptLanguage: string;
```

Comment:
I think those changes shouldn't be here.

Reply:
Oh, thanks. Whoops!